docs/service-loadbalancer.md

```diff
@@ -0,0 +1,288 @@
+not have the layer 2 subnet limitation, but requires the Node network to
+support BGP.
+
+To learn more about MetalLB concepts and functionalities, you can read the
+[MetalLB documentation](https://metallb.universe.tf/concepts).
+
+### Install MetalLB
+
+You can run the following commands to install MetalLB using the YAML manifests:
+
+```bash
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.11.0/manifests/namespace.yaml
+kubectl apply -f https://raw.githubusercontent.com/metallb/metallb/v0.11.0/manifests/metallb.yaml
+```
+
+The commands will deploy MetalLB of version 0.11.0 into Namespace
+`metallb-system`. You can also refer to this [MetalLB installation guide] for
+other ways of installing MetalLB.
+
+As MetalLB will allocate external IPs for all Services of type LoadBalancer,
+once it is running, the Service external IP management feature of Antrea should
+not be enabled to avoid conflicts with MetalLB. You can deploy Antrea with the
+default configuration (in which the `ServiceExternalIP` feature gate of
+`antrea-agent` is set to `false`). MetalLB can work with both `AntreaProxy` and
+`kube-proxy` configurations of `antrea-agent`.
+
+### Configure MetalLB with layer 2 mode
+
+MetalLB is configured through a ConfigMap. To configure MetalLB to work in the
+layer 2 mode, you just need to provide the IP ranges to allocate external IPs.
+For example:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: default
+      protocol: layer2
+      addresses:
+      - 10.10.0.2-10.10.0.10
+```
+
+### Configure MetalLB with BGP mode
+
+The BGP mode of MetalLB requires more configuration parameters to establish BGP
+peering to the router. The example below configures MetalLB using AS nubmer
+64500 to peer with a router at 10.0.0.1 with AS number 64501:
+
+```yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    peers:
+    - peer-address: 10.0.0.1
+      peer-asn: 64501
+      my-asn: 64500
+    address-pools:
+    - name: default
+      protocol: bgp
+      addresses:
+      - 10.10.0.2-10.10.0.10
+```
+
+In addtion to the basic layer 2 and BGP mode configurations described in this
+document, MetalLB supports a few more advanced BGP configurations, and supports
+configuring multiple IP pools which can use different modes. For more
+information, please refer to the [MetalLB configuration guide](https://metallb.universe.tf/configuration).
+
+### Usage
+
+MetalLB will automatically allocate external IPs for every Service of type
+LoadBalancer, and it sets the allocated IP to the `loadBalancer.ingress` field
+in the Service resource `status`. MetalLB also supports user specified `loadBalancerIP`
+in the Service spec. For more information, please refer to the [MetalLB documentation](https://metallb.universe.tf/usage).
```

Comment:
I am not very sure about why this is a separate section called `Usage`. Typically `Usage` is where you explain how to configure things, which you already do above.

Maybe add this information at the beginning of the parent section, before the `### Install MetalLB` subsection.

Reply:
Moved accordingly. Thanks.